composer.json

```
@@ -4,7 +4,7 @@
     "name": "flowpack/restrictcreation",
     "license": "MIT",
     "require": {
-        "neos/neos": "^3.0|^4.0|^5.0|^7.0"
+        "neos/neos": "^3.0|^4.0|^5.0|^7.0|^8.0"
```

This should be changed to
```suggestion
        "neos/neos": "^8.0"
```
I suppose?